server/server_test.go

```diff
@@ -48,6 +49,16 @@ func TestOpenFGAServer(t *testing.T) {
 		ds := memory.New(telemetry.NewNoopTracer(), 10, 24)
 		test.RunAllTests(t, ds)
 	})
+
+	t.Run("TestMySQLDatastore", func(t *testing.T) {
```

Comment:
😬 I'd leave this for a quick PR and then rebase.

Reply:
Sure! I'll go ahead and do that instead.

Reply:
https://github.com/openfga/openfga/pull/281 - will rebase once that's merged.